src/wasm/wasm-type.cpp

```diff
@@ -228,7 +228,7 @@ namespace {
 
 HeapTypeInfo* getHeapTypeInfo(HeapType ht) {
   assert(!ht.isBasic());
-  return (HeapTypeInfo*)ht.getID();
+  return (HeapTypeInfo*)(ht.getRawID());
```

Comment:
I don't feel strongly but `getRawID` could maybe be `getPointer`.

Reply:
What do you think about `getDefinitionID` or `getBaseID`?

Reply:
Hmm, no real preference between those. Current name is fine.